.github/workflows/main.yml

```diff
@@ -65,7 +66,11 @@ jobs:
         cd bufr
         mkdir build
         cd build
-        cmake -DCMAKE_INSTALL_PREFIX=./install -DENABLE_PYTHON=ON ..
+        if [[ ${{ matrix.os }} == "ubuntu-20.04" ]]; then
```

Comment:
I'm not sure I understand the point of this new `if-then-else` switch, given that the action (i.e. `cmake -DCMAKE_INSTALL_PREFIX=./install -DENABLE_PYTHON=ON ..` is the same regardless of the value of the test (??)

Reply:
Yes, I'm introducing the changes piecemeal, because when I put them all in at once the CI failed for some seemingly unrelated reason. ;-) So WTF is going on? I don't know...

This will take me some tinkering to get working, so this is a draft PR. When I get it working I'll mark it ready for review.